tests/modules/rules_webprofiler_test/rules_webprofiler_test.module

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains rules_webprofiler_test.module
+ */
+
+/**
+ * Implements hook_init().
+ *
+ * Each page request it executes test action method to put logs to webprofiler.
+ */
+function rules_webprofiler_test_init() {
```

Comment:
I'm confused - hook_init() does not exist anymore? https://www.drupal.org/node/2013014